test/promql/tests/data_plane/pipeline_to_snapshot_test.yaml

```diff
@@ -19,7 +19,7 @@ tests:
     - series: 'integration_svc_response_seconds_bucket{le="+Inf", namespace="integration-service", source_cluster="cluster02"}'
       values: '0+100x10'  # 100 total occurrences in cluster2
   alert_rule_test:
-    - eval_time: 10m
+    - eval_time: 15m
```

Comment:
I think we should also add a test case in which we test the entire span of the window + the "for".

I.e. a test case in which we evaluate in 21 minutes and we have a bucket which started misbehaving at minute 7, and another that started misbehaving in the same way at minute 0 and we get the alert for the second one and not for the first.